tests/translator/test_arn_generator.py

```diff
@@ -0,0 +1,34 @@
+from unittest import TestCase
```

Comment:
This file has moved to `tests/unit/translator/test_arn_generator.py`

Reply:
The file didn't move but looks like there are two. SAM has some weird testing structures.